.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - run: npm ci
       - run: npx lockfile-lint --path package-lock.json --allowed-schemes "https:" "git+https:" --allowed-hosts npm github.com
       - run: npm run check
-      - run: npm run cover:unit
+      - run: npm run test-coverage-run
```

Comment:
This change is not the same as before. Before we ran unit test, then we deploy and then we run integration and e2e. Now we run all tests and then deploy. Please check.

Reply:
Yes, you are right. I did it on purpose.
Deploying before running tests is not correct I feel.
Why do you need to deploy before running IT and e2e tests?

Reply:
Because you want to test against your newest changes, not against older version. Probably you don't know, but the tests in CI are running against the deployed serverless deployment in GCP (https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/test/test-utils.js#L25).

So your change means that you will run your code against old version in GCP and then deploy the changes after the tests to GCP.

Reply:
I'm not sure if I understand you correctly. Can't you just move this line after deployment is done to line 51?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/976/commits/f3a9bf03b3ff79af16a3d3976dce3a0a64886fe8